src/PointwiseFunctions/GeneralRelativity/WeylMagnetic.hpp

```diff
@@ -0,0 +1,117 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace gr {
+
+//@(
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the magnetic part of the Weyl tensor.
+ *
+ * \details Computes the magnetic part of the Weyl tensor \f$B_{ij}\f$
+ * as: \f$ B_{ij} =
+ \left(1/\sqrt{\det\gamma}\right)D_{k}K_{l(i}\gamma_{j)m}\epsilon^{mlk}
```

Comment:
This line should have a `*` and space like the others. I suspect this is why the documentation check failed. If it still fails after putting the asterisk and space in here and on all future liens of this comment, check that each opening \f$ is closed by a matching \f$. That's what the documentation check failure below is complaining about